src/ruby/spec/generic/rpc_server_pool_spec.rb

```diff
@@ -55,15 +55,15 @@
     it 'it stops being ready after all workers jobs waiting or running' do
       p = Pool.new(5)
       p.start
-      job = proc { sleep(3) } # sleep so workers busy when done scheduling
+      job = proc { sleep(5) } # sleep so workers busy when done scheduling
```

Comment:
I just changed that one to 5 seconds too after changing the other one to 5 seconds. But I guess this is probably an unnecessary wait. I can change this back to 3